lib/order.rb

```diff
@@ -0,0 +1,38 @@
+class Order
+
+  attr_accessor :current_order, :total
+
+  def initialize(menu)
+    @current_order = {}
+    @total = 0
+    @dishes = menu.dishes
+    @sum = 0
+  end
+
+  def add_to_order(dish, quantity)
+    raise 'Dish not available: not part of the menu' unless @dishes.include?(dish)
+    @current_order.merge!({ dish => quantity })
```

Comment:
Performance/RedundantMerge: Use `@current_order[dish] = quantity` instead of `@current_order.merge!({ dish => quantity })`. (https://github.com/JuanitoFatas/fast-ruby#hashmerge-vs-hash-code)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.